src/framework/ui/iuiactionsregister.h

```diff
@@ -27,7 +27,7 @@
 #include "iuiactionsmodule.h"
 #include "uitypes.h"
 #include "async/channel.h"
-
+#include <unordered_map>
```

Comment:
Ah sorry... I left this here by mistake, will remove it next time. Anyways have to squash commits later on